src/test/scala/com/bwsw/cloudstack/vault/server/cloudstack/TestData.scala

```diff
@@ -29,13 +29,12 @@ import com.bwsw.cloudstack.vault.server.common.WeightedQueue
 import scala.util.Random
 
 trait TestData {
-  val userId: UUID = UUID.randomUUID()
   val accountId: UUID = UUID.randomUUID()
   val vmId: UUID = UUID.randomUUID()
   val domainId: UUID = UUID.randomUUID()
 
 
-  val listUsersCommand = "listUsers"
+  val listAccountsCommand = "listAccounts"
   val listVms = "listVirtualMachines"
```

Comment:
is it a command too? as the above variable

Reply:
it is not necessary, removed